app/boards/arm/nrf52840_m2/board.cmake

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: MIT
+
+board_runner_args(nrfjprog "--nrf-family=NRF52" "--softreset")
+
+include(${ZEPHYR_BASE}/boards/common/blackmagicprobe.board.cmake)
```

Comment:
Does this come w/ the adafruit nrf52 bootloader by default? Seems to based on the UF2 build stuff. Likely, we should ditch the blackmagicprobe, and just leave nrfjprog as the option here, since eventually we'll likely want to move to adafruit-nrfjprog as our other flash utility option.

Reply:
No, but adding blackmagic probe will not affect anything else. 
I feel like we can add any number of supported runners here(also in other boards), so that the end user need not edit the board file, in case he use the different supported runner.

As a firmware, we should not ditch any hardware support, thereby not forcing users to use same hardware.
But as for default, we can make the nrfjproj (and later adafruit-nrfjprog) as the first runner so that it will be used as default.
Does this make sense ?

Reply:
Yeah, this makes sense to me. Thanks!